src/core/ext/transport/binder/transport/binder_transport.cc

```diff
@@ -370,6 +370,19 @@ class MetadataEncoder {
 }  // namespace
 }  // namespace grpc_binder
 
+static void accept_stream_locked(void* gt, grpc_error_handle /*error*/) {
+  grpc_binder_transport* gbt = static_cast<grpc_binder_transport*>(gt);
+  if (gbt->accept_stream_fn) {
+    gpr_log(GPR_INFO, "Accepting a stream");
```

Comment:
Nit: Since this is codepath is executed for every RPC call, consider `GPR_DEBUG` instead?

Reply:
This doesn't really expose debugging details like pointer address so I would give it `GPR_INFO`. Besides, `GPR_INFO` won't be printed by default anyway.